src/geneweaver/client/cli/alpha/parse/utils.py

```diff
@@ -61,12 +64,18 @@ def get_metadata(file_path: Path, sheet: Optional[str] = None) -> None:
 
     if file_type == "csv":
         header, header_idx = csv.get_headers(file_path)
-        _print_metadata_csv(file_path, header_idx)
+        try:
+            metadata = general.read_metadata(file_path, header_idx)
+        except (EmptyFileError, ValueError, UnsupportedFileTypeError) as e:
+            print(e)
+            raise typer.Exit(code=1) from e
+
+        print_metadata_csv(file_path, metadata)
     elif file_type == "xlsx":
-        sheet_names, sheet_metadata, _, _, n_sheets = _get_metadata_xlsx(
+        sheet_names, sheet_metadata, _, _, n_sheets = xlsx.get_metadata(
             file_path, sheet
         )
-        _print_metadata_xlsx(file_path, sheet_names, sheet_metadata, n_sheets)
+        print_metadata_xlsx(file_path, sheet_names, sheet_metadata, n_sheets)
 
```

Comment:
Is it worth having that if type is xls then raise an exception? I assume that is rooted out already.

Reply:
Or just else: raise an exception.